lighthouse-core/gather/driver.js

```diff
@@ -1080,17 +1123,18 @@ class Driver {
    * possible workaround.
    * Resolves on the url of the loaded page, taking into account any redirects.
    * @param {string} url
-   * @param {{waitForFcp?: boolean, waitForLoad?: boolean, waitForNavigated?: boolean, passContext?: LH.Gatherer.PassContext}} options
+   * @param {{waitForFcp?: boolean, waitForLcp?: boolean, waitForLoad?: boolean, waitForNavigated?: boolean, passContext?: LH.Gatherer.PassContext}} options
    * @return {Promise<{finalUrl: string, timedOut: boolean}>}
    */
   async gotoURL(url, options = {}) {
     const waitForFcp = options.waitForFcp || false;
+    const waitForLcp = options.waitForLcp || false;
```

Comment:
I having trouble finding where we set this to `true`. Were there changes to gather-runner to enable this that I'm missing? :)

Reply:
Thanks for catching this, this actually explains alot. I thought this PR was working even though there was no LCP lifecycle event, turns out `waitForLcp` just wasn't enabled 😬